### PR TITLE
Feature/UPPSF-666 adding orbs and snyk

### PR DESCRIPTION
**Pull request by gkazakov111**

### CODE REVIEW

File: service/broader_test.go [hunk 1]
```diff
@@ -58,7 +58,7 @@ func TestBroaderConceptsProvider_CheckHealthErrorOnNewRequest(t *testing.T) {
 	checkResult, err := suggester.Check().Checker()
 
 	expect.Error(err)
-	assert.Equal(t, "parse ://__gtg: missing protocol scheme", err.Error())
+	assert.Equal(t, "parse \"://__gtg\": missing protocol scheme", err.Error())
```
Comment (mchompalova) [inline service/broader_test.go:61]: Please remove the string comparison and use either assert.Error() or errors.As() to test the type of error.

File: service/concordance_test.go [hunk 2]
```diff
@@ -53,7 +54,7 @@ func TestConcordanceService_CheckHealthErrorOnNewRequest(t *testing.T) {
 	checkResult, err := suggester.Check().Checker()
 
 	expect.Error(err)
-	assert.Equal(t, "parse ://__gtg: missing protocol scheme", err.Error())
+	assert.Equal(t, "parse \"://__gtg\": missing protocol scheme", err.Error())
```
Comment (mchompalova) [inline service/concordance_test.go:57]: Same as above.

File: service/suggester_test.go [hunk 3]
```diff
@@ -227,7 +227,7 @@ func TestAuthorsSuggester_CheckHealthErrorOnNewRequest(t *testing.T) {
 	checkResult, err := suggester.Check().Checker()
 
 	expect.Error(err)
-	assert.Equal(t, "parse ://__gtg: missing protocol scheme", err.Error())
+	assert.Equal(t, "parse \"://__gtg\": missing protocol scheme", err.Error())
```
Comment (mchompalova) [inline service/suggester_test.go:230]: Same as above.

File: service/suggester_test.go [hunk 4]
```diff
@@ -269,7 +269,7 @@ func TestOntotextSuggester_GetSuggestionsErrorOnNewRequest(t *testing.T) {
 
 	expect.Nil(suggestionResp.Suggestions)
 	expect.Error(err)
-	expect.Equal("parse ://content/suggest: missing protocol scheme", err.Error())
+	expect.Equal("parse \"://content/suggest\": missing protocol scheme", err.Error())
```
Comment (mchompalova) [inline service/suggester_test.go:272]: Same as above.

File: service/suggester_test.go [hunk 5]
```diff
@@ -367,7 +367,7 @@ func TestOntotextSuggester_CheckHealthErrorOnNewRequest(t *testing.T) {
 	checkResult, err := suggester.Check().Checker()
 
 	expect.Error(err)
-	assert.Equal(t, "parse ://__gtg: missing protocol scheme", err.Error())
+	assert.Equal(t, "parse \"://__gtg\": missing protocol scheme", err.Error())
```
Comment (mchompalova) [inline service/suggester_test.go:370]: Same as above.

**Comment by gkazakov111** [comment]
Close this PR, because in another one for structured logging all these changes are presented.

**Pull request closed without merging**